### ISSUE TITLE
SimpleMemoryCache in multiproces environment

### ISSUE THREAD

**Issue by ChronoDK** [aio-libs/aiocache#553]
It seems this doesn't work. Is there a way to make it work, or do I have to use one of the external cache solutions? Or does another similar library exist that allows simple memory caches to be shared between processes?

**Comment by brondsem** [comment]
There is one at https://github.com/luizalabs/shared-memory-dict#aiocache-backend

I have not used it, so I cannot say how well it works etc, but I just found it so thought I'd mention it

**Comment by Dreamsorcerer** [comment]
Great, maybe we can just link to this project in the documentation somewhere and close the issue.

**Comment by Dreamsorcerer** [comment]
Yep, if someone can create a page/section in the docs titled `Other backends` or similar, we can just link to it there.
